prisma/schema.prisma

```diff
@@ -149,3 +151,34 @@ model DeliveryLogLine {
   createdAt DateTime    @default(now())
   updatedAt DateTime    @updatedAt
 }
+
+model Conversation {
+  id        Int         @id @default(autoincrement())
+  name      String
+  createdBy Int
+  messages  Message[]
+  createdAt DateTime    @default(now())
+  updatedAt DateTime    @updatedAt
+  users     UsersAndConversations[]
+}
+
+model Message {
+  id                Int           @id @default(autoincrement())
+  createdBy         String
+  content           String
+  userId            Int
+  user              User          @relation(fields: [userId], references: [id])
+  conversationId    Int
+  conversation      Conversation  @relation(fields: [conversationId], references: [id])
```

Comment:
Let's cascade a delete on the conversation - if a conversation is deleted, so too should the corresponding messages.